### DISCUSSION TITLE
This page is taking too long to load.

### DISCUSSION THREAD

**Discussion by JchanCode** [community/community#35225] [Other Feature Feedback, Questions, & Ideas]
Im getting *Unicorn* error whenever I try to connect my account to third party services. ( ex. aws, Vercel...)
Any insight as to whats happening and how I can fix this would be great! Thank you!

**Comment by 3kh0** [comment]
This happens every so often, here are a few quick fixes that sometimes help.

- Reload the page (easiest in the book)
- Reload and clear cache, in chrome do ctrl+shift+r
- Disconnect from wifi, and reconnect
- Open cmd and do `ipconfig/release` and then `ipconfig/renew`
- Turn off your computer, wait 10 seconds, and turn it back on.
- Try a different browser
- Clear cookies and site data from github.com
- Reboot your router

These are some quick and easy fixes, this issue happens from time to time again, and these fixes seem to fix it for me.

*If this helped please mark this as the answer!* ❤️

**Reply by kevinqtogitty** [reply]
I am getting the same issue here with vercel/netlify. Have tried all the above suggestions to no avail. I had no problems last week deploying an app

**Comment by WalberMelo** [comment]
I am facing the same issue with vercel/netlify. The suggestions above do not work. It seems a general problem

**Reply by Noshi96** [reply]
https://github.com/orgs/community/discussions/34751#discussioncomment-3797859
It worked for me.

**Comment by kevinqtogitty** [comment]
I found a solution, at least for Vercel. 

When you get to the app permissions page, at the very bottom click the "Import Third-party Git Repository". And all you ahve to do is go to the repo you want to connect and copy the HTTPS link and voila. You just need to give a new Repo name and it will copy over the content from your original repo

**Reply by mr-palindrome** [reply]
I think I found the solution. Try leaving those organisation where you are collaborator!
It worked for me!
